zebrad/src/components/mempool/downloads.rs

```diff
@@ -84,14 +84,15 @@ pub enum DownloadAction {
 }
 
 /// A gossiped transaction, which can be the transaction itself or just its ID.
+#[derive(Debug)]
 pub enum GossipedTx {
     Id(UnminedTxId),
     Tx(UnminedTx),
 }
```

Comment:
A nit: because this may be an id or a tx, but an id is not the same as a tx, so maybe:

```suggestion
#[derive(Debug)]
pub enum Gossip {
    Id(UnminedTxId),
    Tx(UnminedTx),
}
```

Reply:
Done in c1c1150